sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -858,19 +858,19 @@ class BigQueryStreamingInsertsErrorHandling(unittest.TestCase):
   # to determine error types and messages to try for retriables.
   @parameterized.expand([
       param(
-          exception_type=exceptions.Forbidden,
+          exception_type=exceptions.Forbidden if exceptions else None,
           error_reason='rateLimitExceeded'),
       param(
-          exception_type=exceptions.DeadlineExceeded,
+          exception_type=exceptions.DeadlineExceeded if exceptions else None,
           error_reason='somereason'),
       param(
-          exception_type=exceptions.ServiceUnavailable,
+          exception_type=exceptions.ServiceUnavailable if exceptions else None,
           error_reason='backendError'),
       param(
-          exception_type=exceptions.InternalServerError,
+          exception_type=exceptions.InternalServerError if exceptions else None,
           error_reason='internalError'),
       param(
-          exception_type=exceptions.InternalServerError,
+          exception_type=exceptions.InternalServerError if exceptions else None,
```

Comment:
Do these tests still run if empty params are passed in? Would it be more optimal to skip them entirely? Something like this in the beginning of each affected test:
```
if exceptions is None:
  self.skipTest("Import fail: google.api_core")
```

Reply:
it's already there: https://github.com/apache/beam/blob/541dfb38105b2453665f9d16f2423456f043168d/sdks/python/apache_beam/io/gcp/bigquery_test.py#L851

this pr fixes the error thrown even when the test is marked as skipped.

Reply:
Oh I see. This LGTM then.